spec/tools/benchmarking/refresh_manager.rb

```diff
@@ -0,0 +1,56 @@
+# Runs a refresh of 1 specified manager
+#
+# Requirements:
+# 1. Add 'manageiq_performance' gem into your dev gemfile
+# 2. Add OpenShift gem as a plugin
+# 3. Add OpenShift provider into your ManageIQ
+#
+# Run from the the root ManageIQ repository as:
+# bundle exec rails r plugins/manageiq-providers-openshift/spec/tools/benchmarking/refresh_manager.rb <manager_name>
+#
+# Parameters:
+# ARGV[0] - name of the manager
+# ARGV[1] - "true" if we want disconnect all the data from the manager, which simulates the first refresh
+# ARGV[2] - "true" if we want to run also profiling, results wil lbe stored under tmp dir
+
+require 'manageiq_performance'
+
+manager_name   = ARGV[0] || "10.16.31.50"
+disconnect_all = ARGV[1] == "true"
+profile        = ARGV[2] == "true"
+
+def refresh(manager_name)
+  ems = ExtManagementSystem.find_by(:name => manager_name)
+
+  _, timings = Benchmark.realtime_block(:ems_total_refresh) do
+    EmsRefresh.refresh(ems)
+  end
+
+  timings
+end
+
+# Show all the SQL queries on the STDERR, will show them even if we redirect the output with >
+ActiveRecord::Base.logger = Logger.new(STDERR)
```

Comment:
since these are hacky scripts, my option is comment/uncomment them :-D 

Though I haven't noticed much of the time difference, the logging itself moves in a different order of magnitude than the rest of the processing (at least while having NVMe SSD :-))